src/DS2431.h

```diff
@@ -0,0 +1,28 @@
+// 0x2D  1Kb 1-Wire EEPROM
+// tested on buspirate and two different real 1-wire masters
+
+#ifndef ONEWIRE_DS2431_H
+#define ONEWIRE_DS2431_H
+#define CRC_INIT 0xCFB5
```

Comment:
using a #define as include-guard is ok for now (i will switch to "#pragma once" later). 
the constant crc_init should be a constexpr. it has advantages like typesafety, no name collisions and more helpful compiler-debug-output